python/lsst/pipe/base/script/transfer_from_graph.py

```diff
@@ -92,18 +97,25 @@ def transfer_from_graph(
 
     # Filter the refs based on requested dataset types.
     filtered_refs = filter_by_dataset_type_glob(output_refs, dataset_type)
+    _LOG.verbose("After filtering by dataset_type, number of datasets to transfer: %d", len(filtered_refs))
 
     dest_butler = Butler.from_config(dest, writeable=True)
 
-    transferred = dest_butler.transfer_from(
-        qbb,
-        filtered_refs,
-        transfer="auto",
-        register_dataset_types=register_dataset_types,
-        transfer_dimensions=transfer_dimensions,
-        dry_run=dry_run,
-    )
-    count = len(transferred)
+    # For faster restarts, filter out those the destination already knows.
+    filtered_refs = filter_by_existence(dest_butler, filtered_refs)
+
+    # Transfer in chunks
+    count = 0
```

Comment:
Can you do something like:
```python
n_chunks = math.ceil(len(refs) / chunk_size)
chunk_num = 0
for chunk in ...:
    chunk_num += 1
    if n_chunks > 1:
        _LOG.verbose("Transferring %d datasets in chunk %d/%d", len(chunk), chunk_num, n_chunks)
    ...
```
so that when people are doing a lot of this we can see the progress? The alternative is to use the progress bar stuff but hardly anyone turns on the progress bar flag and that's not really suitable for batch anyhow.